1.7/administration/installing/custom/advanced.md

```diff
@@ -62,7 +62,7 @@ The DC/OS installation creates these folders:
 
     **Important:** The IP address of a node must not change after DC/OS is installed on the node. For example, the IP address must not change when a node is rebooted or if the DHCP lease is renewed. If the IP address of a node does change, the node must be wiped and reinstalled.
 
-    **Important:** The IP detection scripts must begin with `#!/bin/sh` or `#!/usr/bin/env bash` as shown in the examples below. Do not add any comments before that.
+    **Important:** The IP detection script needs to be `UTF-8` encoded and have a valid [Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line as shown in the examples below.
```

Comment:
also i have changed the whole thing to 
`Create an IP detection script for your environment and save as genconf/ip-detect. This script needs to be UTF-8 encoded and have a valid shebang line. You can use the examples below.`

verify the correctness and tell me.

thanks